ipdb/__main__.py

```diff
@@ -87,6 +87,12 @@ def wrap_sys_excepthook():
 
 
 def set_trace(frame=None, context=3):
+    try:
+        # Try to get and redeclare context with value of env variabe
+        context = os.environ['IPDB_CONTEXT_SIZE']
+    except KeyError:
+        pass
```

Comment:
Yes, You are right, it will be more flexible =) But I made it in other way (I made commit).